src/main/java/com/entropyteam/entropay/employees/repositories/PtoRepository.java

```diff
@@ -14,4 +15,8 @@ public interface PtoRepository extends BaseRepository<Pto, UUID> {
             "ORDER BY ABS(EXTRACT(EPOCH FROM (start_date - CURRENT_TIMESTAMP))) " +
             "LIMIT 1", nativeQuery = true)
     LocalDate findNearestPto(@Param("employeeId") UUID employeeId);
+
+    @Query(value = "SELECT DISTINCT extract('Year' FROM start_date) AS year FROM pto WHERE deleted=false "
+            + " ORDER BY year ASC", nativeQuery = true)
+    List<Integer> getPtosYears();
```

Comment:
It looks like an overly complicated way to get the years. Selecting the year of the first PTO should be enough